src/store/audio.ts

```diff
@@ -1271,7 +1297,10 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
           length += m.consonantLength != undefined ? m.consonantLength : 0;
           length += m.vowelLength;
         });
-        length += phrase.pauseMora ? phrase.pauseMora.vowelLength : 0;
+        if (phrase.pauseMora != null && query.pauseLengthScale != undefined) {
+          const pauseLength = phrase.pauseMora.vowelLength;
+          length += pauseLength * query.pauseLengthScale;
+        }
```

Comment:
`pauseLengthScale`は必須になったので、前のコードを変更する形で分かりやすい書き方ができそう？
```suggestion
        length += phrase.pauseMora
          ? phrase.pauseMora.vowelLength * query.pauseLengthScale
          : 0;
```